joern-cli/frontends/csharpsrc2cpg/src/main/scala/io/joern/csharpsrc2cpg/astcreation/AstForExpressionsCreator.scala

```diff
@@ -48,6 +49,13 @@ trait AstForExpressionsCreator(implicit withSchemaValidation: ValidationMode) {
     astForNode(parenExpr.json(ParserKeys.Expression))
   }
 
+  private def astForCoalesceExpression(coalesceExpression: DotNetNodeInfo): Seq[Ast] = {
+    val leftAst  = astForExpression(createDotNetNodeInfo(coalesceExpression.json(ParserKeys.Left)))
+    val rightAst = astForExpression(createDotNetNodeInfo(coalesceExpression.json(ParserKeys.Right)))
+
+    leftAst ++ rightAst
+  }
```

Comment:
I'd suggest placing its arguments inside an `<operator>` call of some sorts or, alternatively, the ternary `leftAst != null ? leftAst : rightAst`.

Otherwise we won't have their context.